tests/integration/conftest.py

```diff
@@ -450,3 +449,41 @@ def pytest_configure(config):
     config.addinivalue_line(
         "markers", "smoke: mark test as part of smoke test suite"
     )
+
+
+@pytest.fixture
+def create_linode():
+    timestamp = str(time.time_ns())
+    label = "cli" + timestamp
+    result = (
+        exec_test_command(
+            LINODE_BASE_CMD
+            + [
+                "create",
+                "--type",
+                "g6-nanode-1",
+                "--region",
+                "us-ord",
+                "--image",
+                DEFAULT_TEST_IMAGE,
+                "--label",
+                label,
+                "--root_pass",
+                DEFAULT_RANDOM_PASS,
+                "--text",
+                "--delimiter",
+                ",",
+                "--no-headers",
+                "--format",
+                "label,region,type,image,id",
+                "--no-defaults",
+            ]
+        )
+        .stdout.decode()
+        .rstrip()
+    )
+
+    res_arr = result.split(",")
+    linode_id = res_arr[4]
+    yield linode_id
+    delete_target_id(target="linodes", id=linode_id)
```

Comment:
Any usage for this function? I think `create_linode` has been implemented in somewhere else?

Reply:
it was defined in conftest with - def linode_with_label(): and using of i.e 
yield result, linode_id so that new function won't needed 
youjung suggested to have separate fixture and 
one more function is implemented in linode_types with has limitation for package